### ISSUE TITLE
Using motion sensors with temp sensors

### ISSUE THREAD

**Issue by briandmiller** [SANdood/Green-Smart-HW-Recirculator#3]
Can I use a motion sensor to turn pump on and then temp sensor to turn pump off so that pump only runs when motion sensor/s are active and temp is below a setpoint?

**Comment by SANdood** [comment]
Yes, you can select that the Thermometer is used to Turn Off the switch when the temperature is reached, but not to turn it on. 

 

Likewise, you can set it to turn on when motion is detected, but not to turn it off when the motion stops.
